CMakeDoxygenFilter.cpp

```diff
@@ -413,10 +413,10 @@ class CMakeParser
     _os << funcName << '(';
     if (nextToken() == CMakeLexer::TOK_IDENTIFIER)
     {
-      _os << _lexer.getIdentifier();
+      _os << "CMAKE_VARIABLE " << _lexer.getIdentifier();
```

Comment:
A similar change was discussed in issue #1 but not included for the stated reasons. Do you have additional input to this discussion?